test/python/expected/upgrade_validation/expected_dependency.out

```diff
@@ -498,6 +498,7 @@ Function sys.inject_fault(text,integer,integer)
 Function sys.inject_fault_all()
 Function sys.inject_fault_status(text)
 Function sys.int2_fixeddecimal_cmp(smallint,sys.fixeddecimal)
+Function sys.int2_numeric_cmp(smallint,numeric)
```

Comment:
We need a int2 to numeric = condition view test case to avoid add this dependency

Reply:
Added